manifests/08-prometheus_rule.yaml

```diff
@@ -20,4 +20,13 @@ spec:
       for: 5m
       labels:
         severity: info
+    - alert: SCANotAvailable
```

Comment:
I would be in favor of a more meaningful name. Without prior knowledge of the insights operator, SCA is cryptic to me.

Reply:
HI @simonpasquier thanks for reviewing this one. OK. Do you have any suggestion please? e.g `SimpleContentAccessNotAvailable`?

Reply:
OK let me update it..thank you